src/sass/_add-item-view.scss

```diff
@@ -0,0 +1,81 @@
+@use '_variables.scss' as *;
+
+.add-item-container {
+	position: relative;
+	align-items: center;
+	margin-bottom: 4em;
+
+	form {
+		.input {
+			display: flex;
+			flex-direction: column;
+			width: 75%;
+			margin: auto;
+		}
+
+		.input::after {
+			position: absolute;
+			top: 18%;
+			right: 4%;
+			content: url('/img/after.svg');
+		}
+
+		.item-urgency-container {
+			display: flex;
+			flex-direction: column;
+			padding: 30px 10px 0px 10px;
+
+			fieldset {
+				border: 0;
+				display: flex;
+				flex-direction: row;
+				flex-wrap: wrap;
+				justify-content: center;
+				padding: 30px;
+
+				input[type='radio'] {
+					display: none;
+					&:not(:disabled) ~ label {
+						cursor: pointer;
+					}
+				}
+
+				input[type='radio']:checked + label {
+					background: $purple;
+					color: $light-green;
+					box-shadow: 0px 0px 20px $yellow;
+				}
+
+				legend {
+					font-family: $header-font;
+					font-size: 2rem;
+					color: #ff404a;
+				}
+
+				label {
```

Comment:
For accessibility we should put a hover & focus state on these labels, so the user knows which one they're interacting with. And if we put a transition on the parent, it will be a smooth change.
```
input[type='radio']:checked + label {
	background: $purple;
	color: $light-green;
	border-color:$red-dark;
	// box-shadow: 0px 0px 20px $yellow;
}

label {
	padding: 0.7em 1.1em;
	margin: 0 0.3em 0.2em 0.3em;
	border-radius: 15px;
	display: flex;
	align-items: center;
	background: $yellow;
	color: $brown;
	border: 3px solid transparent;
	// box-shadow: 0px 3px 10px -2px hsla(150, 5%, 65%, 0.5);
	position: relative;
	&:hover,
	&:focus {
		border-color:$purple;
		background-color: $purple;
		color: $light-green;
	}
}
```

I'm personally not the biggest fan of box-shadow, but what does everyone else think? I'm happy to include if everyone likes it!

Reply:
The addition of a hover & focus state on the labels would be a great touch! And I also would lean towards no box-shadow but I also don't mind to include it if you choose to. 😁

Reply:
I'm not married to box-shadow. Away it goes! In with the border.

Reply:
No box shadow and no css animation :( I was convinced after researching more on CSS introducing accessibility challenges...